applications/trilinos_application/custom_utilities/parallel_fill_communicator.h

```diff
@@ -390,11 +390,11 @@ class ParallelFillCommunicator
             }
 
         // Create the colored graph for communication.
-        matrix<int> domains_colored_graph;
+        DenseMatrix<int> domains_colored_graph;
         int max_color_found = -1;
         if (my_rank == root_id)
         {
-            matrix<int> domains_graph = scalar_matrix<int>(num_processors, num_processors, 0);
+            DenseMatrix<int> domains_graph = scalar_matrix<int>(num_processors, num_processors, 0);
```

Comment:
The scalar_matrix here is also an ublas type, but I'm unsure what should replace it...